src/components/signoff/SignoffToolBar.tsx

```diff
@@ -110,10 +110,12 @@ export default function SignoffToolBar({
   const { status } = source;
 
   const canRequestReview = canEdit && isEditor(source, sessionState);
+
   const canReview =
     canEdit &&
-    isReviewer(source, sessionState) &&
-    !hasRequestedReview(source, sessionState);
+    ((isReviewer(source, sessionState) &&
+      !hasRequestedReview(source, sessionState)) ||
+      !sessionState.serverInfo?.capabilities?.signer?.to_review_enabled);
```

Comment:
If we want to be thorough, we may want to align this test with the possibility of the server. 

Indeed, the global `to_review_enabled` can be true at the server level (which is the case of the DEV server for example), and be set to false for a specific collection. 
For example, in our prod config we have:
```ini
    ### Normandy recipes
    # Only the Normandy server is allowed to publish recipes. Multi-signoff is disabled.
    kinto.signer.main-workspace.normandy-recipes.to_review_enabled = false
    kinto.signer.main-workspace.normandy-recipes.group_check_enabled = false
    kinto.signer.main-workspace.normandy-recipes-capabilities.to_review_enabled = false
    kinto.signer.main-workspace.normandy-recipes-capabilities.group_check_enabled = false
```
which then ends up in `serverInfo.capabilities.signer.resources[{i}].to_review_enabled` as you can see here https://remote-settings.allizom.org/v1/ 
```
"signer": {
      "description": "Digital signatures for integrity and authenticity of records.",
      "url": "https://github.com/Kinto/kinto-signer#kinto-signer",
      "version": "32.1.0",
      "resources": [
        {
          "source": {
            "bucket": "staging",
            "collection": null
          },
          "destination": {
            "bucket": "blocklists",
            "collection": null
          },
          "preview": {
            "bucket": "blocklists-preview",
            "collection": null
          }
        },
        {
          "source": {
            "bucket": "main-workspace",
            "collection": "normandy-recipes"
          },
          "destination": {
            "bucket": "main",
            "collection": "normandy-recipes"
          },
          "to_review_enabled": false
        },
```

Also I wouldn't test for falsy, I would test that the `to_review_enabled` is explicitly set to false

Reply:
Ooo, didn't realize there was a possibility of multiple layers here.

A big follow up question (since I don't see existing unit tests verifying this functionality). If I have defined values for `to_review_enabled` at both the server level and collection level, which one "wins"? Does the collection level always win? Or if either are `false`, do we always consider it disabled?

Reply:
So, if we want the client behaviour to match the server's, then _collection overrides the server_.

I don't think it would affect your code, but note that in practice, we use it only in one way: the server has review enabled, and we disable it for certain collections. We don't use the opposite, like for example on the dev server review is globally disabled but we don't enable it for certain collections.

Reply:
Got it. So putting this into pseudo code logic:
```
review_is_enabled = server_to_review_enabled && collection_to_review_enabled
```

Reply:
> `review_is_enabled = server_to_review_enabled && collection_to_review_enabled`
>

Yes, except that `collection_to_review_enabled` is implicitly true

Reply:
I _think_ I've captured this logic in the latest changes.